CHANGES.rst

```diff
@@ -39,6 +39,8 @@ Bug Fixes
 Cubeviz
 ^^^^^^^
 
+- Future proof slicing logic for as_steps implementation in upcoming glue-jupyter release. [#1599]
```

Comment:
I am not sure if this needs a change log. User is not impacted yet, so this might just confuse them.

Reply:
I wasn't sure where to put this, but my thinking was that jdaviz 2.9 (the latest release) will break for anyone who updates glue-jupyter to 1.3 (once that is released)... so it will _become_ a bugfix to 2.9 once that happens.  If someone stays on 2.9, updates glue-jupyter, and sees the break, then maybe this entry will tell them that an update to 2.10 will fix their problem?  But I also see your point, that at the time 2.10 is released, the bug doesn't actually exist yet.

Reply:
Maybe more visible as a known issue entry then? "If you have this version but upgraded that package, this thing will break" under the Installation section.

Reply:
except we can't go back and add that to the 2.9 docs afaik, which is where it would be really useful.  In the future, especially once we pin glue-jupyter, that entry in the stable docs will be irrelevant.

Reply:
I think our RTD defaults to `latest`, not `stable`.

But if you insist on the change log, then that is fine too. Mine was merely a suggestion.